modules/compute/gke-node-pool/outputs.tf

```diff
@@ -65,9 +65,14 @@ output "tolerations" {
   value       = local.tolerations
 }
 
-output "instructions" {
-  description = "Instructions for submitting the sample GPUDirect enabled job."
-  value       = <<-EOT
+locals {
+  gpu_direct_enabled = var.machine_type == "a3-highgpu-8g" || var.machine_type == "a3-megagpu-8g"
+  script_path = {
+    a3-highgpu-8g = "enable-tcpx-in-workload.py",
+    a3-megagpu-8g = "enable-tcpxo-in-workload.py"
+  }
+  gpu_direct_instruction = <<-EOT
+    Since you are using a3-mega machine type, you will have to add gpudirect support to your workload.
```

Comment:
nit: Make this `a3-mega` or `a3-high` based on the machine type ? or maybe just add the use `machine_type `